### PR TITLE
MODSOURMAN-380: Expand journalRecord entity with "title" property.

### PR DESCRIPTION
**Pull request by VRohach**
1. "title" field added to the "journalRecord.json".

### CODE REVIEW

**Review by RuslanLavrov — APPROVED** [review]

File: schemas/mod-source-record-manager/journalRecord.json [hunk 1]
```diff
@@ -68,6 +68,10 @@
       "description": "Error message",
       "type": "string"
     },
+    "title": {
+      "description": "Journal record title",
```
Comment (RuslanLavrov) [inline schemas/mod-source-record-manager/journalRecord.json:72]:
```suggestion
      "description": "Title from source record",
```

**Review by KaterynaSenchenko — APPROVED** [review]
please add sample entity as well

File: schemas/mod-source-record-manager/journalRecord.json (hunk 1 above)
Comment (VRohach) [reply schemas/mod-source-record-manager/journalRecord.json]: Done.

**Comment by VRohach** [comment]
> please add sample entity as well

Done

**Pull request merged**